webapp/utils/user_agent.jsx

```diff
@@ -81,6 +81,10 @@ export function isMobileApp() {
     return isAndroid() || isIos();
 }
 
+export function isMobile() {
```

Comment:
Oh, I'm not smart enough to understand my own function names apparently. I assumed `isMobileApp` wasn't checking for mobile browsers for some reason. I'll fix that.